core/src/main/java/org/elasticsearch/index/query/GeoDistanceQueryBuilder.java

```diff
@@ -76,12 +75,7 @@
     /** Whether or not to accept geo points with invalid latitude or longitude */
     private boolean ignoreMalformed = DEFAULT_IGNORE_MALFORMED;
 
-    static final GeoDistanceQueryBuilder PROTOTYPE = new GeoDistanceQueryBuilder();
-
-    /** For serialization purposes only. */
-    private GeoDistanceQueryBuilder() {
-        this.fieldName = null;
-    }
+    static final GeoDistanceQueryBuilder PROTOTYPE = new GeoDistanceQueryBuilder("field");
```

Comment:
We don't allow for empty String in most constructors (at least fieldname), whitespace would work but decided against it to make it clearer what the arguments are. Can change this of course if there's other reasons.